site/content/en/docs/drivers/includes/none_usage.inc

```diff
@@ -1,8 +1,11 @@
 ## Requirements
 
-VM running a systemd-based Linux distribution ([see #2704](https://github.com/kubernetes/minikube/issues/2704))
+A Linux VM with the following:
 
-kubeadm requirements listed here - https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/
+* Docker
+* systemd (OpenRC based systems are also supported in v1.10+)
```

Comment:
how about just say systemd or OpenRC. 
to clearify u can add example of distros with OpenRC that u have tesed it on

Reply:
I don't want to just say OpenRC -- since support for it is not available in the current stable release.

Reply:
... also, I used Debian for OpenRC.